build.sbt

```
@@ -26,7 +26,9 @@ inThisBuild(Def.settings(
     Some("scm:git:git@github.com:portable-scala/portable-scala-reflect.git"))),
 ))
 
-lazy val `portable-scala-reflect` = crossProject(JSPlatform, JVMPlatform)
+lazy val `portable-scala-reflect` = crossProject(JSPlatform,
+                                                 JVMPlatform,
+                                                 NativePlatform)
```

I believe keeping this one line is much more readable, especially because it's chained with other method calls:
```suggestion
lazy val `portable-scala-reflect` = crossProject(JSPlatform, JVMPlatform, NativePlatform)
```